test/integration/targets/inventory_docker_machine/playbooks/test_inventory_1.yml

```
@@ -0,0 +1,25 @@
+---
+- hosts: dm-inventory-test
+  gather_facts: no
+  tasks:
+  - name: do something
+    file:
+      path: /tmp/deleteme
+      state: directory
+
+- hosts: localhost
+  environment:
+    DOCKER_CERT_PATH: "{{ hostvars['dm-inventory-test'].dm_DOCKER_CERT_PATH }}"
+    DOCKER_HOST: "{{ hostvars['dm-inventory-test'].dm_DOCKER_HOST }}"
+    DOCKER_MACHINE_NAME: "{{ hostvars['dm-inventory-test'].dm_DOCKER_MACHINE_NAME }}"
+    DOCKER_TLS_VERIFY: "{{ hostvars['dm-inventory-test'].dm_DOCKER_TLS_VERIFY }}"
+  tasks:
+  - name: docker-compose up on target Docker Machine host
+    docker_service:
+      project_name: test
+      definition:
+        version: '2'
+        services:
+          test:
+            container_name: test
+            image: ubuntu/18.04:Latest
```

Shouldn't that be
```suggestion
            image: ubuntu:18.04
```
(https://hub.docker.com/_/ubuntu)?

Also, it's probably better to use a smaller image for testing, like
```suggestion
            image: hello-world:latest
```
That one is tiny :)

---

It should be lower case. The 'latest' tag at the end was fine according to https://hub.docker.com/_/ubuntu, however good idea to use a smaller image.

---

Especially if you don't need anything special about the image, `hello-world` is a very good one since it is so tiny ;) As soon as you need something which runs for some time, there are others (I'm using `alpine:3.8` with command `/bin/sh -c "sleep 10m"` in some other integration tests, in case you ever need something like that).